modules/cart/src/Form/AddToCartForm.php

```diff
@@ -179,7 +179,8 @@ public function buildEntity(array $form, FormStateInterface $form_state) {
     $entity = parent::buildEntity($form, $form_state);
     // Now that the purchased entity is set, populate the title and price.
     $entity->setTitle($entity->getPurchasedEntity()->getLineItemTitle());
-    $entity->setUnitPrice($this->chainPriceResolver->resolve($entity->getPurchasedEntity()));
+    $price = $this->chainPriceResolver->resolve($entity->getPurchasedEntity());
```

Comment:
Wait, it might be. What if people are reacting to the Add to Cart event and want the resolved price?